### PR TITLE
@86eqzwj2x #added security block for settings management

### PR DESCRIPTION
**Pull request by adrian92612**

### CODE REVIEW

File: apps/web-customer/src/components/blocks/HubBlocks/Settings/AccountBlock/AccountBlock.tsx [hunk 1]
```diff
@@ -1,11 +1,22 @@
 import React from "react";
 import { Box } from "@mui/material";
-
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { accountSchema } from "../../../../../core/Schema/SettingsForm/validation";
+import { Button, EvaIcon, TextField } from "core-library/components";
+import Image from "next/image";
+import { profile, google } from "core-library/assets"
 interface AccountProps {
```
Comment (be3jaay) [inline apps/web-customer/src/components/blocks/HubBlocks/Settings/AccountBlock/AccountBlock.tsx:9]: Always add spacing between import and interface.

File: apps/web-customer/src/components/blocks/HubBlocks/Settings/PlansBlock/PlansBlock.tsx [hunk 2]
```diff
@@ -16,21 +24,93 @@ export const PlansBlock: React.FC<PlansProps> = ({ title, subTitle }) => {
...
+          <div className='border-b w-full  border-[#0F2A71] border-opacity-50'>
+            <p className='p-2 font-extrabold px-5'>
+              Subscription Plan
+            </p>
+          </div>
+          <div className=' min-h-[240px] flex w-[100%]'>
+            {plans.length > 0 && plans.map((item) => (
+              <>
+                <PlansCard plan={item} />
+              </>))}
```
Comment (be3jaay) [inline apps/web-customer/src/components/blocks/HubBlocks/Settings/PlansBlock/PlansBlock.tsx:38]: No need for fragment here. just return the PlansCard

File: apps/web-customer/src/components/blocks/HubBlocks/Settings/SecurityBlock/SecurityBlock.tsx [hunk 3]
```diff
@@ -1,13 +1,46 @@
 import React from "react";
 import { Box } from "@mui/material";
+import { TextField } from "core-library/components";
+import { Box, Switch, SxProps } from "@mui/material";
```
Comment (be3jaay) [inline apps/web-customer/src/components/blocks/HubBlocks/Settings/SecurityBlock/SecurityBlock.tsx:4]: We can use our own Switch component from the core-library

File: apps/web-customer/src/components/blocks/HubBlocks/Settings/SecurityBlock/SecurityBlock.tsx [hunk 4]
```diff
@@ -1,13 +1,46 @@
...
+import { Button, EvaIcon } from "core-library/components";
```
Comment (be3jaay) [inline apps/web-customer/src/components/blocks/HubBlocks/Settings/SecurityBlock/SecurityBlock.tsx:5]:
```suggestion
import { 
Button, 
EvaIcon,
TextField,
Switch,
} from "core-library/components";
```

File: apps/web-customer/src/components/blocks/HubBlocks/Settings/PlansBlock/PlansBlock.tsx [hunk 5]
```diff
@@ -16,21 +24,93 @@ export const PlansBlock: React.FC<PlansProps> = ({ title, subTitle }) => {
...
+                </Box>
+                <Box>
+                  <Typography fontWeight={600}>Visa Ending 3532</Typography>
+                  <Typography >Expiry date: 12/25</Typography>
+                </Box>
+              </Box>
+            </Grid>
+          </Box>
+          <Box sx={{ borderRadius: '15px', width: 1 }} className=" bg-[#f3f4f8]">
+            <div className='border-b w-full  border-[#0F2A71] border-opacity-50'>
```
Comment (be3jaay) [inline apps/web-customer/src/components/blocks/HubBlocks/Settings/PlansBlock/PlansBlock.tsx:88]:
```suggestion
            <div className='border-b w-full  border-darkBlue border-opacity-50'>
```

**Review by be3jaay — CHANGES_REQUESTED** [review]

File: apps/web-customer/src/components/blocks/HubBlocks/Settings/PlansBlock/PlansBlock.tsx (hunk 2 above)
Comment (adrian92612) [reply apps/web-customer/src/components/blocks/HubBlocks/Settings/PlansBlock/PlansBlock.tsx:38]: Noted sir, removed the fragments.

File: apps/web-customer/src/components/blocks/HubBlocks/Settings/SecurityBlock/SecurityBlock.tsx (hunk 3 above)
Comment (adrian92612) [reply apps/web-customer/src/components/blocks/HubBlocks/Settings/SecurityBlock/SecurityBlock.tsx:4]: Noted, used our own SwitchButton.

File: apps/web-customer/src/components/blocks/HubBlocks/Settings/SecurityBlock/SecurityBlock.tsx (hunk 4 above)
Comment (adrian92612) [reply apps/web-customer/src/components/blocks/HubBlocks/Settings/SecurityBlock/SecurityBlock.tsx:5]: Imports updated

**Review by BryanPalad — APPROVED** [review]

**Review by be3jaay — APPROVED** [review]

**Pull request merged**